### ISSUE TITLE
Results and Console Log: Incorrect role is defined for tab controls(Results and console.log) under executed stored procedure 'testcosmosdb' tab section.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1046]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242692

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure.
Navigate to "New Stored Procedure" menu item and select it.
Navigate to Stored Procedure ID edit field and give input and select Save menu item.
Now Navigate to "Execute" control and select it.
Navigate through all the controls under "Input Parameters" blade and select Execute button.
Navigate to all the controls like 'Results' and 'Console.log' tab items.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Incorrect role as a button is defined for tab controls(Results and console.log) under executed stored procedure 'testcosmosdb' tab section.

Expected Result:
Correct role should be defined for tab controls(Results and console.log) under executed stored procedure 'testcosmosdb' tab section as a 'tab' instead of button.
![image](https://user-images.githubusercontent.com/8519407/132716990-5bd22dd3-75c6-483e-a9bd-1cb5a9128e05.png)
